pkgs/development/python-modules/ipympl/default.nix

```
@@ -9,6 +9,8 @@ buildPythonPackage rec {
     sha256 = "f0f1f356d8cb9d4fb51bb86dbbf837c190145316cb72f66081872ebc4d6db0a1";
   };
 
+  buildInputs = [ jupyter-packaging ];
```

```suggestion
  nativeBuildInputs = [ jupyter-packaging ];
```
This is most likely only required for installing.